### DISCUSSION TITLE
scrolling bar-

### DISCUSSION THREAD

**Discussion by Lavishq** [kovidgoyal/kitty#7599] [General]
![image](https://github.com/kovidgoyal/kitty/assets/96654752/02dc9f97-ad00-44de-ab0a-ed3fb458703e)
is it possible to use the scroll using mouse via moving scrollbar? 
if yes, then plz send the link to doc or issue 
if no, why?
the reason for asking is i tried to look up for a while on internet, issues, discussions, docs and couldnt find it

also the reason i would need that is due to the output on terminal will be too large for some programs and using that w/ touchpad will be super convenient and efficient to find what i was looking from the text dump in the terminal :penguin: :penguin:

**Comment by kovidgoyal** [comment]
No, it's not possible. If you want to scroll using mouse, either use two fingers and move them up and down on your touchpad or use a scrollback pager that supports scrolling via mouse.

**Reply by Lavishq** [reply]
yeah, using that but it is too slow process and feel inefficient for large output texts

will it be possible in future to add this feature?

**Reply by Lavishq** [reply]
@kovidgoyal ser, is it impossible to add this scrollable feature?
![image](https://github.com/user-attachments/assets/d12e6c0a-698b-40cb-860b-b5ac0995dc49)
since scrolling these via touchpad takes an eternity i thought i would do a quick follow up just in case :''')

or is there another way to scroll up quickly on kitty

page up/down wont do nor ctrl up/down - atleast something pls which would make it even better so that scrolling 30-40 seconds is not required :'''''')

**Reply by malkir** [reply]
@kovidgoyal it's a fairly glaring issue to have a scrollbar that does not function, I'm not familiar with a single piece of software that does this.

**Comment by neoakris** [comment]
I found this via google search and just wanted to comment that this seems to be fixed, it's just not default.

I followed ben2talk's directions here
https://www.reddit.com/r/KittyTerminal/comments/1m1mlaf/how_to_add_a_scrollbar_to_kitty/

1. Click on Kitty Terminal for Mouse Focus
2. In top left menu of Mac OS I went to kitty -> Settings... 
3. clicking settings popped up vi to edit a kitty config file
I then added this (from the above reddit post), and mouse wheel started working
```
scrollback_lines 2000
scrollback_indicator_opacity 1.0
wheel_scroll_multiplier 5.0
wheel_scroll_min_lines 1
```

**Comment by cooper2k4** [comment]
The feature has been added to kitty v0.43

https://github.com/kovidgoyal/kitty/pull/8945
https://sw.kovidgoyal.net/kitty/changelog/#a-scrollbar-for-the-kitty-scrollback-0-43
